pkg/app/bottomBar.go

```diff
@@ -185,7 +184,7 @@ func (bb *BottomBar) Render() {
 	gl.BindVertexArray(bb.textVaoID)
 	gl.EnableVertexAttribArray(0)
 	gl.EnableVertexAttribArray(1)
-	bb.font.Bind()
+	gl.BindTexture(gl.TEXTURE_2D, bb.fontInfo.TextureID())
```

Comment:
also here

Reply:
It doesn't seem right to have one opengl call placed in a different file in our mess of opengl calls. We could have more OpenGL abstractions and make it all function calls so we are less prone to make mistakes, but I think for now we should keep it in one place